testing/fuchsia/run_tests.py

```diff
@@ -1,12 +1,28 @@
-#!/usr/bin/env python3
+#!/usr/bin/env vpython3
+
+# [VPYTHON:BEGIN]
+# python_version: "3.8"
+# wheel <
+#   name: "infra/python/wheels/pyyaml/${platform}_${py_python}_${py_abi}"
+#   version: "version:5.4.1.chromium.1"
+# >
+# [VPYTHON:END]
+
 # Copyright (c) 2013, the Flutter project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be found
 # in the LICENSE file.
 
 import argparse
+import logging
 import os
 import sys
 
+from subprocess import CompletedProcess
+from typing import List
+
+# The import is coming from vpython wheel and pylint cannot find it.
+import yaml  # pylint: disable=import-error
```

Comment:
@dnfield do you know how we would go about adding yaml to the engine deps?

Reply:
FYI: this part is copied from https://cs.opensource.google/flutter/recipes/+/master:recipe_modules/yaml/resources/parse_yaml.py?q=import.yaml&ss=flutter